Composer/packages/lib/shared/src/appschema.ts

```diff
@@ -1960,7 +1960,7 @@ export const appschema: OBISchema = {
           description: 'Time out setting for the OAuth signin card.',
           default: '900000',
         },
-        tokenProperty: {
+        property: {
           $role: 'expression',
           title: 'Token property',
```

Comment:
Do the other properties need to change? Like title, description, etc?

Reply:
There are some additional schema like allowInteruption etc, let's don't change it for R8, because it needs UX update which is too heavy.